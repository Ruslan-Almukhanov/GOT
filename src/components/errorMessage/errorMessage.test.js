import React from 'react'
import ErrorMessage from './errorMessage';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';

describe('Component <errorMessage/> rendered correctly', () => {
    it('rendered correctly', () => {
        const errorMessage = render(<ErrorMessage />)
        expect(errorMessage).toMatchSnapshot();
    })
    it('image corectly rendered', () => {
        const { getByTestId } = render(<ErrorMessage />)
        expect(getByTestId('img-error')).toBeTruthy()
    })
})