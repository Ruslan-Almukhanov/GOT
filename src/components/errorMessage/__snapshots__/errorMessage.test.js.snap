// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component <errorMessage/> rendered correctly rendered correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <img
        class="sc-bwzfXH boSKPF"
        data-testid="img-error"
        src="/img/error.jpg"
      />
      <span
        class="sc-bdVaJa cUIpFB"
      >
         something go wrong
      </span>
    </div>
  </body>,
  "container": <div>
    <img
      class="sc-bwzfXH boSKPF"
      data-testid="img-error"
      src="/img/error.jpg"
    />
    <span
      class="sc-bdVaJa cUIpFB"
    >
       something go wrong
    </span>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
